.ci/Jenkinsfile

```diff
@@ -7,12 +7,14 @@ pipeline {
     REPO = 'apm-agent-nodejs'
     BASE_DIR = "src/github.com/elastic/${env.REPO}"
     PIPELINE_LOG_LEVEL='INFO'
-    NOTIFY_TO = credentials('notify-to')
     JOB_GCS_BUCKET = credentials('gcs-bucket')
     GITHUB_CHECK_ITS_NAME = 'Integration Tests'
     ITS_PIPELINE = 'apm-integration-tests-selector-mbp/main'
     OPBEANS_REPO = 'opbeans-node'
     GITHUB_CHECK = 'true'
+    RELEASE_URL_MESSAGE = "(<https://github.com/elastic/${env.REPO}/releases/tag/${env.TAG_NAME}|${env.TAG_NAME}>)"
+    SLACK_CHANNEL = '#apm-agent-nodejs'
```

Comment:
That's right, I didn't enable notifications from the PullRequest but from Tags, I'm gonna proceed with a new release simulation that should also send notifications, if no issues from your side.

As stated in https://github.com/elastic/apm-agent-nodejs/pull/2585#issuecomment-1055320796, I'd like to give another go, therefore the new ARNs will be ending in `2`.

Reply:
![image](https://user-images.githubusercontent.com/2871786/156208305-eb708160-f4b2-45d5-968b-a0401b49209d.png)

(bear in mind, the existing null references are expected since it uses a specific env variable only available when running a build from a Git tag)